taxonomies/additives.txt

```diff
@@ -8714,32 +8714,29 @@ wikidata:en:Q17997701
 
 en:E336, Potassium tartrates
 xx:E336
-bg:E336, E336 food additive
-ca:E336, Tartrat de potassi
-cs:E336, E336 food additive
-da:E336, E336 food additive
-de:E336, Kaliumtartrate
-el:E336, E336 food additive
+bg:E336, Натриеви тартарати
+ca:E336, Tartrats de sodi
```

Comment:
All the added translations are for sodium tartrate (E335), not potassium tartrate (E336)